src/components/notification-icon.component.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Text, View, StyleSheet } from 'react-native';
+import { Icon } from 'react-native-elements';
+
+import { colors, fonts } from 'config';
+
+const styles = StyleSheet.create({
+  badge: {
+    position: 'absolute',
+    right: 1,
+    top: -1,
+    backgroundColor: colors.darkerRed,
+    borderRadius: 18,
+    width: 18,
+    height: 18,
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderColor: colors.alabaster,
+    borderWidth: 1,
+  },
+  badgeText: {
+    alignSelf: 'center',
+    fontSize: 9,
+    ...fonts.fontPrimary,
+    color: colors.white,
+    backgroundColor: 'transparent',
+  },
+});
+
+const mapStateToProps = state => ({
+  notificationsCount: state.notifications.notificationsCount,
+});
+
+class NotificationIconComponent extends Component {
+  props: {
+    iconColor: string,
+    notificationsCount: string,
+  };
+
+  render() {
+    const { iconColor, notificationsCount } = this.props;
+
+    return (
+      <View>
+        <Icon
+          containerStyle={{ justifyContent: 'center', alignItems: 'center' }}
+          color={iconColor}
+          name="notifications"
+          size={33}
+        />
+
+        {notificationsCount
+          ? <View style={styles.badge}>
```

Comment:
Hey @lex111, maybe we could use <Badge> from react-native-elements which is already bundled : 
  https://react-native-training.github.io/react-native-elements/API/badge/

Reply:
It's not round, I like that currenly it's just round in shape, well okay

Reply:
Fair enough, if you would rather have a circular badge and think it looks neater I don't mind :)

I think we can do one of two things however:

1. Could we have a local badge component that contains the style for the badge and expose a background color and text color prop? Feels nice to have that exposed in it's own component. Would prefer if you call it `circular-badge` or something along those lines to not name-conflict with the react native elements badge component
2. OR we _could_ use React Native Elements badge here and just apply `containerStyle` to make it circular.

I'm okay with either option :raised_hands:

Reply:
Strangely, I can not import this component:

```js
import { Icon, Badge } from 'react-native-elements';

console.log(Badge); // undefined
```

I do not understand what's up, okay, I'll make a local component then.